A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -105,6 +105,14 @@ var A320_Neo_UpperECAM;
                 return -1;
             }
         }
+        gearNotDownLanding() {
+            const rh = SimVar.GetSimVarValue("RADIO HEIGHT", "feet") <= 750;
+            const flapIndex = Simplane.getFlapsHandleIndex();
+            const gearNotDwnLocked = SimVar.GetSimVarValue("GEAR POSITION", "percent") <= 99;
+            const N1Low = SimVar.GetSimVarValue("ENG N1 RPM:1", "percent") <= 75 || SimVar.GetSimVarValue("ENG N1 RPM:2", "percent") <= 75;
```

Comment:
Use `this.getCachedSimVar` instead of `SimVar.GetSimVarValue` here.

Reply:
Resolved.